### PR TITLE
Update readme example

### PR DESCRIPTION
**Pull request by yordis**
Related to #414

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,15 +52,26 @@ defmodule MyApp.Guardian do
   use Guardian, otp_app: :my_app
 
   def subject_for_token(resource, _claims) do
-    {:ok, to_string(resource.id)}
+    # You could send any information you want
```
Comment (doomspork) [inline README.md]: Can we reword this a little differently?  How about:

> You can use any value for the subject of your token but it should be useful in retrieving the resource later.  A unique `id` is a good subject, a non-unique email address is a poor subject.

File: README.md [hunk 2]
```diff
@@ -52,15 +52,26 @@ defmodule MyApp.Guardian do
...
+    {:ok, sub}
   end
-
   def subject_for_token(_, _) do
     {:error, :reason_for_error}
   end
 
   def resource_from_claims(claims) do
-    {:ok, find_me_a_resource(claims["sub"])}
+    # In this case `claims["sub"]` is the `id of the resource`
```
Comment (doomspork) [inline README.md]: How about:

> Here we'll look our resource from the claims, the subject can be found in the `"sub"` key.  In above `subject_for_token/2` we returned the resource `id` so here we'll rely on that to look it up.

File: README.md [hunk 3]
```diff
@@ -52,15 +52,26 @@ defmodule MyApp.Guardian do
...
+    # because we previously used the `id` of the resource for `sub`
+    # of the tokens see `subject_for_token` function.
+    id = claims["sub"]
+    resource = MyApp.get_resource_by_id(id)
+
+    # Do whatever you want for find the resource
```
Comment (doomspork) [inline README.md]: Let's remove this comment.  I think this is applied in the previous comments.

Comment (yordis) [reply README.md]: @doomspork at this point I am commenting for kids (no offenses but people don't like to read and study anymore, they just want to copy + paste)

File: README.md [hunk 4]
```diff
@@ -52,15 +52,25 @@ defmodule MyApp.Guardian do
...
+    {:ok, sub}
   end
-
   def subject_for_token(_, _) do
     {:error, :reason_for_error}
   end
 
   def resource_from_claims(claims) do
-    {:ok, find_me_a_resource(claims["sub"])}
+    # Here we'll look our resource from the claims, the subject can be
```
Comment (doomspork) [inline README.md]: Sorry @yordis, I had a typo in my previous suggestion.  It should be "look up"

**Comment by doomspork** [comment]
Thank you @yordis 👍

**Pull request merged**